spec/human_enum_spec.rb

```diff
@@ -34,6 +34,10 @@
         include_examples 'translations', :direction,
                          north: 'Up?', south: 'South', east: 'East', west: 'West'
       end
+
+      context 'with multiple models using the same enum name' do
+        include_examples 'multiple models'
+      end
     end
```

Comment:
suggestion: I think, in this case, it makes more sense to omit the `context` block and use `it_behaves_like` instead:
```suggestion
      it_behaves_like 'multiple models using the same enum name'
```